Casks/t/texifier.rb

```diff
@@ -8,15 +8,18 @@
   homepage "https://www.texifier.com/mac"
 
   livecheck do
-    url :homepage
-    regex(/href=["'].*?download\.texifier\.com.*?osx.*?Texifier_(\d+(?:_\d+)+)__(\d+)__([^_]+)\.dmg/i)
-    strategy :page_match do |page, regex|
-      page.scan(regex).map { |match| "#{match[0].tr("_", ".")},#{match[1]},#{match[2]}" }
+    url "https://www.texifier.com/apps/updates/texifier/appcast-stable.xml"
+    regex(/Texifier[._-]v?(\d+(?:_\d+)+)__(\d+)__([^_]+)\.dmg/i)
```

Comment:
```suggestion
    regex(/Texifier[._-]v?(\d+(?:[._]\d+)+)__(\d+)__(\h+)\.dmg/i)
```

* When the primary version text uses a delimiter other than `.` like `_`, I tend to use `[._]`, so the regex would continue to match if upstream switches to a typical dot delimiter.
* The last capture group is a hash (e.g., `f7f8226`, `38a73af`, etc.), so we can use `\h+` there.

Reply:
Thanks!